build/circle-test.sh

```diff
@@ -95,7 +100,12 @@ time ${builder} make GITHOOKS= testrace \
 # acceptance tests on the Mac's, but circle-deps.sh only built the
 # acceptance tests for Linux.
 if [ "$(uname)" = "Linux" ]; then
-  time $(dirname $0)/../acceptance.test -test.v -test.timeout 5m
+  time $(dirname $0)/../acceptance.test -test.v -test.timeout 5m --verbosity=1 --vmodule=monitor=2 | \
+    tr -d '\r' | tee "${outdir}/acceptance.log" | \
+    grep -E "^\--- (PASS|FAIL)|^(FAIL|ok)|${match}" |
```

Comment:
The same `grep` line is used or test, testrace and acceptance. Perhaps make the regexp a variable.